setup.py

```diff
@@ -0,0 +1,36 @@
+from setuptools import setup, find_packages
+
+setup(
+	name='aloception',
+	author='Visual Behavior',
+	version='9.6.22',
```

Comment:
Why this version number ?

Reply:
Today's date, because there are no `version.txt` file yet

Reply:
Maybe you should use the version of the current release of aloception.